src/sagemaker_training/mpi.py

```diff
@@ -28,25 +28,31 @@
 logger = logging_config.get_logger()
 logging.getLogger("paramiko").setLevel(logging.INFO)
 
-exception_classes = None
-try:
-    from smdistributed.modelparallel.backend import exceptions
 
-    # list of exceptions SMMP wants training toolkit to catch and log
-    exception_classes = [x for x in dir(exceptions) if isclass(getattr(exceptions, x))]
-except ImportError:
-    logger.info("No exception classes found in smdistributed.modelparallel.backend")
+def set_exception_classes():
+    """Set exception classes"""
```

Comment:
can you rename this to `get_modelparallel_exception_classes` as it helps better understand this function? @haohanchen-yagao

Reply:
Updated the function name